firstvoices/backend/tests/test_apis/test_song_api.py

```diff
@@ -36,7 +124,7 @@ def create_instance_with_media(
             related_videos=related_videos,
         )
 
-    def get_expected_list_response_item(self, song, site):
+    def get_expected_list_response_item_summary(self, song, site):
```

Comment:
I don't see where this is used. Are you maybe missing a test case to go with it?

Reply:
yep, missed a commit there.